### PR TITLE
Move fixtable to npm

### PR DESCRIPTION
**Pull request by smargovsky**
📢  Fixtable core is now on NPM!

This PR copies much of the work @toddjordan did w/ moving font-awesome to npm.  Blame him if anything is wrong ;)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,6 +59,7 @@
     "ember-cli-htmlbars": "^1.1.1",
     "ember-cli-htmlbars-inline-precompile": "^0.3.3",
     "ember-truth-helpers": "1.2.0",
+    "fixtable": "^2.0.1",
```
Comment (menewman) [inline package.json]: Would this be a good time to bump to fixtable 2.0.2? It looks like Mike patched a few days ago.

File: index.js [hunk 2]
```diff
@@ -34,10 +22,16 @@ module.exports = {
       parent.import(path.join('vendor', 'fonts', fontFileName));
     });
 
-    importFixtable(parent);
+    let fixtablePath = path.join(this.nodeModulesPath, 'fixtable/dist/');
+
+    parent.import({
+      development: path.join(fixtablePath, 'fixtable.js'),
+      production: path.join(fixtablePath, 'fixtable.min.js')
```
Comment (menewman) [inline index.js]: Since we're moving this into our build tree, do we need to include the minimized JS file here, or should we just include the full thing?

**Comment by menewman** [comment]
I think we also need to change the documentation in the README to remove all references to Bower. Consumers no longer need to be concerned about pulling in Bower deps, which means we can remove all the stuff about blueprints.

Speaking of which, it looks like you already updated the addon's blueprint, which is great. Actually, though, I think we can just delete the blueprint entirely now, since it doesn't do anything other than pull in Bower deps.

File: index.js (hunk 2 above)
Comment (smargovsky) [inline index.js]: Done

File: package.json (hunk 1 above)
Comment (smargovsky) [inline package.json]: Done

**Comment by smargovsky** [comment]
Should we also remove .bowerrc, .bower.json, and bower from the .travis.yaml files?

**Comment by menewman** [comment]
I think we should keep those because we could conceivably use Bower again in the future (although hopefully not). Those generic Bower files are part of all default Ember addons/apps, IIRC, so they shouldn't be a problem.

**Comment by menewman** [comment]
Looks good -- thanks @smargovsky!

**Pull request merged**